tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Builders/OrderSummaryModelBuilder.cs

```diff
@@ -10,13 +11,15 @@ internal sealed class OrderSummaryModelBuilder
         private readonly string _description;
         private Guid _organisationId;
         private IEnumerable<SectionModel> _sections;
+        private readonly string _status;
```

Comment:
Are there no unit tests to go along with this change?

Reply:
Yeah I looked at this. Quite a few tests were failing but by adding the default value they then passed. So I checked there were tests covering this functionality, but didn't add a specific test as we currently don't have the ability to change that status yet as per a user case. Don't mind either way, would you like me to add a specific test?